hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -374,16 +374,44 @@
       tests: []
     }
 
-    // PINMUX (pre-verified IP) integration tests:
+    // PINMUX & PADRING (pre-verified IP) integration tests:
     {
-      name: chip_sw_pin_mux
+      name: chip_pin_mux
       desc: '''Verify the MIO muxing at input and output sides.
 
-            SW programs MIO INSEL and OUTSEL CSRs to connect and verify each muxed source. At the
-            moment, GPIOs are the only mux inputs.
+            - Enable `stub_cpu` mode.
+            - Add a forcing interface to pinmux's pad-facing DIO and MIO ports, including the output
+              enables; and a sampling interface for the peripheral facing DIO and MIO ports.
+            - Similarly, add a driving / sampling interface for all DIOs and MIOs at the chip pads.
+            - In the output direction:
+              - Program all MIO outsel and pad attribute registers to random values.
+              - Force the pad-facing pinmux MIO ports and output enables to random values.
+              - Vrify all MIO pad values for correctness.
+            - For the input direction:
+              - Program all MIO insel and pad attribute registers to random values.
+              - Drive the MIO pads to random values.
+              - Probe and sample the peripheral facing MIO ports of the pinmux and verify the values
+                for correctness.
+            - Follow a similar testing procedure for DIOs.
```

Comment:
Missing `'''` to end the heredoc? this seems to be causing a CI failure.

Reply:
Fixed.